raycicmd/converter.go

```diff
@@ -181,6 +181,22 @@ func (c *converter) convertPipelineGroup(g *pipelineGroup) (
 	}
 
 	for _, step := range g.Steps {
+		// filter steps by tags
+		if step_tags, ok := step["tags"]; ok && tagFilters != nil {
+			skip := true
+			for _, tag := range step_tags.([]interface{}) {
```

Comment:
could you put this for loop into a util function and add some simple unit tests?

I am also thinking maybe also accept tags as a space delimited string, and you can covert this into a set of strings from different types.